### DISCUSSION TITLE
Metal cannot be set on iOS and falls back to automatic (OpenGL)

### DISCUSSION THREAD

**Discussion by alixdotsh** [ppy/osu#23792] [Q&A] [answered]
### Type

Performance

### Bug description

trying to change automatic to Metal fails upon restart, and goes back to automatic, which seems to be OpenGL. This happens every time I have tried to change it.

This might be different since I had installed the new laser build via [altstore](http://altstore.io), but I doubt that could be the cause.

as for device specs, I have been using an iPad Pro M1 with iOS 16.4.1 (a)

### Screenshots or videos

![IMG_1865](https://github.com/ppy/osu/assets/58949994/20a05914-6b36-4a7f-a049-124d90090f81)

### Version

2023.605.0

### Logs

can't grab them on iOS

**Comment by peppy** [comment]
On an iPad Pro (not M1) metal still works fine. We'll need more feedback from someone else with one of the M1 iPads.

That said, do note that we absolutely cannot support lazer installs from third party sources. We have no idea what binary you have installed and what they've done to it.

**Reply by alixdotsh** [reply]
The binary was untouched. I had downloaded the IPA from the latest release, all it does is install it to my system and makes it able to be run. I only did this as a last resort because TestFlight seemed to be reset.

**Reply by Tobi-kun** [reply]
I have 2020 iPad Pro 11 with A12Z Bionic chip, I have the same problem. I just got into the testflight beta program. After selecting Metal and choosing restart, the screen goes black as if quitting, but stays black. After I manually quit osu! in recent apps and run it again it shows the same message. (edit: I'll provide logs as soon as I have time to install itunes on my pc)

**Reply by peppy** [reply, marked as answer]
This will be fixed in the next release, which should arrive in the next day or two. Thanks for the reports.
